amp-rnaseq_reprocessing/amp-rnaseq_reprocess-workflow/test-data/directory-to-files_one-level.yaml

```diff
@@ -0,0 +1,3 @@
+dir:
+  class: Directory
+  path: ./test-dir/nested-dir
```

Comment:
This is the first of the "test-data" files. These are mostly all arguments files used to test a particular tool or workflow in different ways -- see `test-descriptions.yaml` below. Also, let me know if you want to talk over how cwltest works. It's not well documented.